// br-mask/__ivy_ngcc__/dist/app.module.js
import { CommonModule } from '@angular/common';
import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
import { BrMaskDirective } from './directives/br-mask';
import * as ɵngcc0 from '@angular/core';
var BrMaskerModule = (function () {
    function BrMaskerModule() {
    }
BrMaskerModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: BrMaskerModule });
BrMaskerModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function BrMaskerModule_Factory(t) { return new (t || BrMaskerModule)(); }, imports: [[
            CommonModule
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(BrMaskerModule, { declarations: function () { return [BrMaskDirective]; }, imports: function () { return [CommonModule]; }, exports: function () { return [BrMaskDirective]; } }); })();
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(BrMaskerModule, [{
        type: NgModule,
        args: [{
                declarations: [
                    BrMaskDirective
                ],
                exports: [
                    BrMaskDirective
                ],
                imports: [
                    CommonModule
                ],
                schemas: [
                    CUSTOM_ELEMENTS_SCHEMA
                ]
            }]
    }], function () { return []; }, null); })();
    return BrMaskerModule;
}());
export { BrMaskerModule };
/** @nocollapse */
BrMaskerModule.ctorParameters = function () { return []; };
